File: src/main/java/com/jinhe/modules/system/entity/SysAppResource.java
package com.jinhe.modules.system.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

import java.io.Serializable;
import java.time.LocalDateTime;

/**
 * <p>
 * 
 * </p>
 *
 * @author rls
 * @since 2020-11-27
 */
@ApiModel(value="SysAppResource对象", description="")
public class SysAppResource implements Serializable {

    private static final long serialVersionUID=1L;

    private String id;

    @ApiModelProperty(value = "父建id")
    private String parentId;

    @ApiModelProperty(value = "英文名称")
    private String name;

    @ApiModelProperty(value = "显示名称")
    private String displayName;

    @ApiModelProperty(value = "描述")
    private String note;

    @ApiModelProperty(value = "状态0正常1禁用")
    private Integer state;

    @ApiModelProperty(value = "类型（预留）")
    private Integer type;

    @ApiModelProperty(value = "排序")
    private Integer sort;

    @ApiModelProperty(value = "小图标")
    private String icon;

    @ApiModelProperty(value = "创建者id")
    private String createUserId;

    @ApiModelProperty(value = "创建时间")
    private LocalDateTime createTime;

    @ApiModelProperty(value = "更新")
    private LocalDateTime updateTime;


    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getParentId() {
        return parentId;
    }

    public void setParentId(String parentId) {
        this.parentId = parentId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDisplayName() {
        return displayName;
    }

    public void setDisplayName(String displayName) {
        this.displayName = displayName;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }

    public Integer getState() {
        return state;
    }

    public void setState(Integer state) {
        this.state = state;
    }

    public Integer getType() {
        return type;
    }

    public void setType(Integer type) {
        this.type = type;
    }

    public Integer getSort() {
        return sort;
    }

    public void setSort(Integer sort) {
        this.sort = sort;
    }

    public String getIcon() {
        return icon;
    }

    public void setIcon(String icon) {
        this.icon = icon;
    }

    public String getCreateUserId() {
        return createUserId;
    }

    public void setCreateUserId(String createUserId) {
        this.createUserId = createUserId;
    }

    public LocalDateTime getCreateTime() {
        return createTime;
    }

    public void setCreateTime(LocalDateTime createTime) {
        this.createTime = createTime;
    }

    public LocalDateTime getUpdateTime() {
        return updateTime;
    }

    public void setUpdateTime(LocalDateTime updateTime) {
        this.updateTime = updateTime;
    }

    @Override
    public String toString() {
        return "SysAppResource{" +
        "id=" + id +
        ", parentId=" + parentId +
        ", name=" + name +
        ", displayName=" + displayName +
        ", note=" + note +
        ", state=" + state +
        ", type=" + type +
        ", sort=" + sort +
        ", icon=" + icon +
        ", createUserId=" + createUserId +
        ", createTime=" + createTime +
        ", updateTime=" + updateTime +
        "}";
    }
}

File: src/main/java/com/jinhe/modules/system/entity/SysAppResource.kt
package com.jinhe.modules.system.entity;

import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty
import java.io.Serializable
import java.time.LocalDateTime
/**
 * <p>
 * 
 * </p>
 *
 * @author rls
 * @since 2020-11-11
 */
@ApiModel(value="SysAppResource对象", description="")
class SysAppResource : Serializable {

    var id: String? = null
    @ApiModelProperty(value = "父建id")
    var parentId: String? = null
    @ApiModelProperty(value = "英文名称")
    var name: String? = null
    @ApiModelProperty(value = "显示名称")
    var displayName: String? = null
    @ApiModelProperty(value = "描述")
    var note: String? = null
    @ApiModelProperty(value = "状态0正常1禁用")
    var state: Int? = null
    @ApiModelProperty(value = "类型（预留）")
    var type: Int? = null
    @ApiModelProperty(value = "排序")
    var sort: Int? = null
    @ApiModelProperty(value = "小图标")
    var icon: String? = null
    @ApiModelProperty(value = "创建者id")
    var createUserId: String? = null
    @ApiModelProperty(value = "创建时间")
    var createTime: LocalDateTime? = null
    @ApiModelProperty(value = "更新")
    var updateTime: LocalDateTime? = null


    override fun toString(): String {
        return "SysAppResourse{" +
        "id=" + id +
        ", parentId=" + parentId +
        ", name=" + name +
        ", displayName=" + displayName +
        ", note=" + note +
        ", state=" + state +
        ", type=" + type +
        ", sort=" + sort +
        ", icon=" + icon +
        ", createUserId=" + createUserId +
        ", createTime=" + createTime +
        ", updateTime=" + updateTime +
        "}"
    }
}
